### PR TITLE
[18.06] vndr of buildkit to 98f1604

### PR DESCRIPTION
**Pull request by andrewhsu**
**- What I did**

update buildkit vndr to 98f1604 and the associated vndrs of the vendor.conf of buildkit

for reference, comparison of upstream changes: https://github.com/moby/buildkit/compare/9acf51e49185b348608e0096b2903dd72907adcb...98f1604134f945d48538ffca0e18662337b4a850

also updated containerd and units to match the git commit hash of the buildkit `vendor.conf`

**- How I did it**

```
$ vi vendor.conf
$ make -f docker.Makefile shell
$ vndr github.com/moby/buildkit
```

**- How to verify it**
```
$ make -f docker.Makefile vendor
...
2018/08/14 06:28:44 Success
2018/08/14 06:28:44 Running time: 1m31.138362914s
scripts/validate/check-git-diff vendor
vendor is correct
```
**- Description for the changelog**

* Update buildkit to 98f1604

**- A picture of a cute animal (not mandatory but encouraged)**

🐍

### CODE REVIEW

File: vendor/github.com/containerd/containerd/vendor.conf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 github.com/containerd/go-runc f271fa2021de855d4d918dbef83c5fe19db1bdd5
-github.com/containerd/console 9290d21dc56074581f619579c43d970b4514bc08
+github.com/containerd/console 5d1b48d6114b8c9666f0c8b916f871af97b0a761
```
Comment (thaJeztah) [inline vendor/github.com/containerd/containerd/vendor.conf]: Perhaps we should update this one as well (could be separate from this PR https://github.com/containerd/console/compare/9290d21dc56074581f619579c43d970b4514bc08...5d1b48d6114b8c9666f0c8b916f871af97b0a761)

File: vendor/github.com/containerd/containerd/vendor.conf [hunk 2]
```diff
@@ -57,22 +58,26 @@ github.com/golang/glog 44145f04b68cf362d9c4df2182967c2275eaefed
 github.com/google/gofuzz 44d81051d367757e1c7c6a5a86423ece9afcf63c
 github.com/hashicorp/errwrap 7554cd9344cec97297fa6649b055a8c98c2a1e55
 github.com/hashicorp/go-multierror ed905158d87462226a13fe39ddf685ea65f1c11f
-github.com/json-iterator/go 1.0.4
-github.com/opencontainers/runtime-tools 6073aff4ac61897f75895123f7e24135204a404d
+github.com/json-iterator/go f2b4162afba35581b6d4a50d3b8f34e33c144682
```
Comment (thaJeztah) [inline vendor/github.com/containerd/containerd/vendor.conf]: hm, this is somewhere between 1.1.3 and 1.1.4; https://github.com/json-iterator/go/compare/f2b4162afba35581b6d4a50d3b8f34e33c144682...1.1.4 - I can open a PR upstream to make it use a tagged release again

File: vendor/github.com/containerd/containerd/vendor.conf [hunk 3]
```diff
@@ -57,22 +58,26 @@ github.com/golang/glog 44145f04b68cf362d9c4df2182967c2275eaefed
...
+github.com/modern-go/reflect2 05fbef0ca5da472bbf96c9322b84a53edc03c9fd
```
Comment (thaJeztah) [inline vendor/github.com/containerd/containerd/vendor.conf]: Same here; this is somewhere between 1.0.0 and 1.0.1 https://github.com/modern-go/reflect2/compare/05fbef0ca5da472bbf96c9322b84a53edc03c9fd...1.0.1 - I can open a PR upstream for that

File: vendor/github.com/containerd/containerd/vendor.conf (hunk 1 above)
Comment (andrewhsu) [reply vendor/github.com/containerd/containerd/vendor.conf]: The docker/cli 18.06 vendor.conf is already updated to the same git commit: https://github.com/docker/cli/blob/bb983ee75b2616f240ff6a4c6745813fbe2661df/vendor.conf#L94

Comment (thaJeztah) [reply vendor/github.com/containerd/containerd/vendor.conf]: 🤦‍♂️ ah, thanks, didn't check that one in the end

**Pull request merged**